Parent/Parent/Course/CourseDetailsViewController.swift

```diff
@@ -120,7 +141,14 @@ extension CourseDetailsViewController: HorizontalPagedMenuDelegate {
         case .grades:
             return NSLocalizedString("Grades", comment: "")
         case .syllabus:
-            return NSLocalizedString("Syllabus", comment: "")
+            switch courses.first?.defaultView {
+            case .wiki:
+                return NSLocalizedString("Homepage", comment: "")
```

Comment:
Checked with Dave, he says the home page tab should be titled based on the content of the home page. So this should be "Front Page".